packages/core/webpack.config.js

```diff
@@ -0,0 +1,45 @@
+const path = require("path");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin")
+
+module.exports = {
+  entry: ["./src/main.js"],
+  output: {
+    path: __dirname + "/dist",
+    filename: "app.bundle.js",
+  },
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        use: {
+          loader: "babel-loader",
+        },
+      },
+      {
+        test: /\.css$/,
+        use: [
+          MiniCssExtractPlugin.loader, // instead of style-loader
+          'css-loader'
+        ]
+      }
+    ],
+  },
+  resolve: {
+    alias: {
+      '@department-of-veterans-affairs/web-components/react-bindings':
+        path.resolve(__dirname, '../web-components/'),
```

Comment:
Fairly straightforward webpack config. This alias is specifically for the `Banner` component where we are using the `VaAlert` react binding.

Reply:
Is the plan to remove this alias later? Why not import the react bindings from `@department-of-veterans-affairs/web-components`, then?

Reply:
I'd like to remove this by way of converting `Banner` to a web component, and that would eliminate this awkward dependency situation.

>  Why not import the react bindings from @department-of-veterans-affairs/web-components

I think that would require further modification to the `web-components` build, and I don't know what all would be required for that. Currently, we have this which gives us that folder to begin with:

https://github.com/department-of-veterans-affairs/component-library/blob/15d606e04ea012c1fb8242bd390103a67010b0e6/packages/web-components/stencil.config.ts#L28-L32

And we're including it in our `files` property in `package.json`, but these other properties mean that `react-bindings` isn't included at the root level:

https://github.com/department-of-veterans-affairs/component-library/blob/15d606e04ea012c1fb8242bd390103a67010b0e6/packages/web-components/package.json#L5-L8

We might be able to achieve a named import at the root level if we had another entry file that rexported the index in `dist` and `react-bindings` and then had that file be the `main`, but that's adding more complexity.